cpp/DBHostObject.cpp

```diff
@@ -238,6 +238,8 @@ void DBHostObject::create_jsi_functions() {
   });
 
   function_map["close"] = HOSTFN("close") {
+    invalidated = true;
```

Comment:
This is not necessarily a problem, but I discovered during hot reloads if the DB was already closed it would try to be invalidated again and sqlite does not like it.